### PR TITLE
ENH: Use AWS storage

### PR DESCRIPTION
**Pull request by Eric89GXL**
Use Amazon AWS for storage of our datasets :)

### CODE REVIEW

**Comment by agramfort** [comment]
did you upload all the data already?

how much data can you store? do you have full write access?

**Comment by Eric89GXL** [comment]
I uploaded the three files that are part of this PR, yeah -- I think those are the only ones at MGH. I have full write access. I can give you write access, too :)

File: mne/datasets/utils.py [hunk 1]
```diff
@@ -119,17 +119,17 @@ def _data_path(path=None, force_update=False, update_path=True, download=True,
...
+        url = 'https://s3.amazonaws.com/mne-python/datasets/' + archive_name
         hash_ = '3e9e83c642136e5b720e2ecc5dcc3244'
     elif name == 'somato':
         archive_name = 'MNE-somato-data.tar.gz'
         folder_name = "MNE-somato-data"
-        url = 'ftp://surfer.nmr.mgh.harvard.edu/pub/data/MNE/' + archive_name
+        url = 'https://s3.amazonaws.com/mne-python/datasets/' + archive_name
         hash_ = 'f3e3a8441477bb5bacae1d0c6e0964fb'
     elif name == 'testing':
         archive_name = 'mne-testing-data-master.tar.gz'
```
Comment (agramfort) [inline mne/datasets/utils.py]: shouldn't you put testing too?

Comment (Eric89GXL) [inline mne/datasets/utils.py]: I don't see much point:

1. It's already hosted on GitHub
2. We don't want to unnecessarily drive up traffic for that dataset on AWS if we don't have to
3. It should be easier to change directly in GitHub

Ideally we could actually just put all of our datasets on GitHub, but we can't due to size limitations

**Comment by agramfort** [comment]
alright !

thx

**Comment by Eric89GXL** [comment]
cc @haribharadwaj we have moved our datasets off of the MGH servers, be happy :)

**Comment by haribharadwaj** [comment]
Yay @Eric89GXL

**Pull request merged**